tests/test_transforms.py

```diff
@@ -117,19 +117,32 @@ def test_tth_to_q_bad(wavelength, tth, expected_error_type, expected_error_msg):
 
 
 @pytest.mark.parametrize(
-    "q, expected_d",
+    "q, expected_d, warning_expected",
```

Comment:
I am thinking having `warning_expected` explicitly provides a quick overview whether each test case provides a warning or not. @sbillinge how do you like it?